Author: Steve Robbins <smr@debian.org>
Bug-Debian: http://bugs.debian.org/554748
Forwarded: no
Last-Update: 2011-03-20
Description: Fix linking of test suite
 Add libgtest.la to LDADD for sample1_unittest and gtest_all_test.

--- gtest-1.5.0.orig/Makefile.am
+++ gtest-1.5.0/Makefile.am
@@ -234,6 +234,7 @@ TESTS += samples/sample1_unittest
 check_PROGRAMS += samples/sample1_unittest
 samples_sample1_unittest_SOURCES = samples/sample1_unittest.cc
 samples_sample1_unittest_LDADD = lib/libgtest_main.la \
+				 lib/libgtest.la \
                                  samples/libsamples.la
 
 # Another sample.  It also verifies that libgtest works.
@@ -247,7 +248,7 @@ samples_sample10_unittest_LDADD = lib/li
 TESTS += test/gtest_all_test
 check_PROGRAMS += test/gtest_all_test
 test_gtest_all_test_SOURCES = test/gtest_all_test.cc
-test_gtest_all_test_LDADD = lib/libgtest_main.la
+test_gtest_all_test_LDADD = lib/libgtest_main.la lib/libgtest.la
 
 # Tests that fused gtest files compile and work.
 FUSED_GTEST_SRC = \
